packages/p2p/src/components/floating-rate/__test__/floating-rate.spec.js

```diff
@@ -39,6 +39,6 @@ describe('<FloatingRate/>', () => {
     it('should render the exchange rate in hint', () => {
         render(<FloatingRate fiat_currency='AED' local_currency='INR' />);
 
-        expect(screen.getByText('1 AED = 100.00 INR')).toBeInTheDocument();
+        expect(screen.getByText('1 AED = 100.00')).toBeInTheDocument();
```

Comment:
took away local currency here as it doesn't need to have the local currency displayed in floating rate field anymore